src/locales/en.json5

```diff
@@ -1,10 +1,42 @@
 // action: This is the text for an action, for example, the text of a button.
 // field: This is the text of a form field.
 // header: This is the text of a table column header.
-
-// tab: This is the text of a navigation tab.
+// tab: This is the text of a navigation tab, which may also be shown as the page title (browser tab).
 
 {
+  // This is the text shown as the title of a page including in the browser tab.
+  "title": {
+    "resetPassword": "Reset Password",
```

Comment:
It'd be great to remove the `title` message from `AccountResetPassword` as well.